packages/realm-react/package.json

```diff
@@ -43,7 +42,7 @@
   },
   "peerDependencies": {
     "react": ">=16.8.0",
-    "realm": ">=10.17.0"
+    "realm": ">=10.17.0 || >=10.20.0-beta"
```

Comment:
```suggestion
    "realm": "^10.17.0 || ^11.0.0-rc"
```

Reply:
Do we need an upper bound?  Would that mean that the next major version would not be compatible?

Reply:
Good point.  Thanks for the feedback.